gulpfile.js

```diff
@@ -210,9 +210,7 @@ function swallowImages() {
         dest: {
             path: 'images',
             filename: 'images.js',
-            header: 'module.exports = { // This file generated by gulp-imagine-64 at '
-            + (new Date).toLocaleTimeString() + ' on '
-            + (new Date).toLocaleDateString() + '\n',
+            header: 'module.exports = {\n',
```

Comment:
Retired the timestamp which was causing unnecessary inclusion in every commit even when images have not changed. (We were not until recently saving this file in the repo.)